openmrs/apps/clinical/app.json

```diff
@@ -36,6 +36,8 @@
                 { "locale" : "pt", "css": "offline-language-portuguese-brazil"}
             ]
         },
+        "tabularViewIgnoreHeadingsList": ["display", "uuid", "image", "$$hashKey", "activeVisitUuid", "hasBeenAdmitted", "forwardUrl", "programUuid", "enrollment"],
```

Comment:
1. keep config names consistent and readable. e.g. "ignoredTabularViewHeadings" - 
2. why is "forwardUrl", "$$hashKey" is this list. they were never shown right? "forwardUrl" is just a link that takes to an URL - is that feature not allowed anymore? - 
3. individual lists can return new columns as well - do we filter out from a common set?

Reply:
1. Will take this change
2. These were the list of items that were ignored in bahmniapps earlier. I think these can be left for implementers to decide whether to include or not because the searchHandler sql query should initially have these headings. Will remove it from here.
3. AFAIK, this is a common list that we use across all the queues that show up in clinical tabular views